src/herder/HerderSCPDriver.h

```diff
@@ -186,8 +189,9 @@ class HerderSCPDriver : public SCPDriver
 
     struct SCPTiming
     {
-        std::optional<VirtualClock::time_point> mNominationStart;
-        std::optional<VirtualClock::time_point> mPrepareStart;
+        std::optional<VirtualClock::time_point> mNominationStart{};
+        std::optional<VirtualClock::time_point> mNominationAccept{};
```

Comment:
It would probably be useful to export a metric for nominate accept - this way we know the delay to trigger.

Reply:
Fixed